### PR TITLE
tools: adding example log for ignoring txn in drainer 

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? (Required)

adding example log for ignoring txn in drainer

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

**Comment by yikeke** [comment]
Is this a special week PR? Did you push to a wrong branch? @WangXiangUSTC

**Comment by WangXiangUSTC** [comment]
> Is this a special week PR? Did you push to a wrong branch? @WangXiangUSTC

not for the special week, can merge it later

File: tidb-binlog/tidb-binlog-faq.md [hunk 1]
```diff
@@ -126,7 +126,13 @@ TiDB 配置开启 `ignore-error` 写 binlog 失败后触发 critical error 告
...
 1. 查看 drainer.log 日志，查找 `exec failed` 找到 Drainer 退出前最后一条执行失败的 DDL。
 2. 将 DDL 改为下游兼容支持的版本，在下游数据库中手动执行。
-3. 查看 drainer.log 日志，查找执行失败的 DDL 语句，可以查询到该 DDL 的 commit-ts。
+3. 查看 drainer.log 日志，查找执行失败的 DDL 语句，可以查询到该 DDL 的 commit-ts。例如：
+
+    ```
+    [2020/05/21 09:51:58.019 +08:00] [INFO] [syncer.go:398] ["add ddl item to syncer, you can add this commit ts to `ignore-txn-commit-ts` to skip this ddl if needed"] [sql="ALTER TABLE `test` ADD INDEX (`index1`)"] ["commit ts"=416815754209656834]。
+    ```
+
+    通过以上日志中 `sql` 记录的语句查找到这行日志，然后获取到 `commit ts` 信息。
```
Comment (scsldb) [inline tidb-binlog/tidb-binlog-faq.md:135]: 这一行有点多余

**Comment by yikeke** [comment]
> > Is this a special week PR? Did you push to a wrong branch? @WangXiangUSTC
> 
> not for the special week, can merge it later

ok

File: tidb-binlog/tidb-binlog-faq.md (hunk 1 above)
Comment (yikeke) [reply tidb-binlog/tidb-binlog-faq.md:135]: PTAL @WangXiangUSTC

**Pull request merged**